src/parse-templates.ts

```diff
@@ -17,11 +17,23 @@ export interface TemplateLiteralMatch {
   end: RegExpMatchArray;
 }
 
+/**
+ * Represents a static import of a template literal.
+ *
+ * @param importPath the path to the package from which we want to import the template literal (e.g.: 'ember-cli-htmlbars')
+ * @param importIdentifier the name of the template literal (e.g.: 'hbs') or 'default' if this package exports a default function
+ */
 export interface StaticImportConfig {
   importPath: string;
   importIdentifier: string;
 }
```

Comment:
Tweak: this is a place to annotate the properties rather than to use `@param`: as is it won't show up in editors etc. in a useful way.

```ts
/**
 * Represents a static import of a template literal.
 */
export interface StaticImportConfig {
  /**
   * The path to the package from which we want to import the template literal
   * (e.g.: 'ember-cli-htmlbars')
   */
  importPath: string;
  /**
   * The name of the template literal (e.g.: 'hbs') or 'default' if this package
   * exports a default function
   */
  importIdentifier: string;
}
```

Reply:
Done. Thanks!